nibabies/workflows/anatomical/segmentation.py

```diff
@@ -15,6 +15,73 @@
 )
 
 from ...config import DEFAULT_MEMORY_MIN_GB
+from ...interfaces.images import RemapLabels
+
+
+def init_aseg_to_mcrib_wf(omp_nthreads=1, name="aseg_to_mcrib_wf"):
```

Comment:
Workflows are generally used for multistep operations - since this workflow only really has 1 node, it can just be converted to a node.